Makefile

```diff
@@ -216,45 +221,65 @@ build-windows:
 
 .PHONY: image-quick
 image-quick:
-	$(DOCKER) build -t $(IMAGE):$(VERSION) --build-arg BASE=scratch .
-	$(DOCKER) build -t $(IMAGE):$(VERSION)-debug --build-arg BASE=gcr.io/distroless/base:debug .
-	$(DOCKER) build -t $(IMAGE):$(VERSION)-rootless --build-arg USER=1000 --build-arg BASE=scratch .
+	$(DOCKER) buildx build -t $(IMAGE):$(VERSION)-linux-amd64 --platform linux/amd64 --build-arg BASE=scratch .
```

Comment:
Following the discussion over on https://github.com/open-policy-agent/gatekeeper/pull/602 it seems like the current thinking is we do not specify a `--platform` here so you can avoid using qemu. Seems like the change is to specify the `FROM --platform=$BUILDPLATFORM` in the dockerfile it will build using the correct base image, then we just need the go build to generate the correct binary.

Reply:
That would work as well